js_modules/dagit/packages/core/src/asset-graph/layout.ts

```diff
@@ -221,13 +222,7 @@ export const extendBounds = (a: IBounds, b: IBounds) => {
   return {x: xmin, y: ymin, width: xmax - xmin, height: ymax - ymin};
 };
 
-export const ASSET_NODE_ICON_WIDTH = 20;
-export const ASSET_NODE_NAME_MAX_LENGTH = 32;
-const DISPLAY_NAME_PX_PER_CHAR = 8.0;
-
-export const assetNameMaxlengthForWidth = (width: number) => {
-  return (width - ASSET_NODE_ICON_WIDTH) / DISPLAY_NAME_PX_PER_CHAR;
-};
+export const ASSET_NODE_NAME_MAX_LENGTH = 28;
```

Comment:
This is based on the 255px width below, minus the box padding, outline, icon, margin from icon to text, etc. Thought about making const vars for all the values, but it's a bunch of layers...